pkg/reconciler/identity/reconciler.go

```diff
@@ -42,33 +41,38 @@ import (
 )
 
 const (
-	Add                          = "add"
-	Remove                       = "remove"
 	Role                         = "roles/iam.workloadIdentityUser"
 	deleteWorkloadIdentityFailed = "WorkloadIdentityDeleteFailed"
 	workloadIdentityFailed       = "WorkloadIdentityReconcileFailed"
-	concurrencyError             = "googleapi: Error 409: There were concurrent policy changes."
 )
 
-// defaultRetry represents that there will be 3 iterations.
-// The duration starts from 100ms and is multiplied by factor 2.0 for each iteration.
-var defaultRetry = wait.Backoff{
-	Steps:    5,
-	Duration: 100 * time.Millisecond,
-	Factor:   2.0,
-	// The sleep at each iteration is the duration plus an additional
-	// amount chosen uniformly at random from the interval between 0 and jitter*duration.
-	Jitter: 1.0,
-}
+// TODO remove global instance
+var (
+	policyManager       *IAMPolicyManager
+	createPolicyManager sync.Once
+)
 
 func NewIdentity(ctx context.Context) *Identity {
+	createPolicyManager.Do(func() {
+		c, err := admin.NewIamClient(context.Background())
```

Comment:
```suggestion
		c, err := admin.NewIamClient(ctx)
```

Reply:
The manager will be shared by other controller so I don't think it should the controller's ctx